data/mods/ssb/scripts.js

```diff
@@ -292,6 +292,13 @@ let BattleScripts = {
 		this.runEvent('TerrainStart', source, source, status);
 		return true;
 	},
+	modifyDamage(baseDamage, pokemon, target, move, suppressMessages) {
```

Comment:
im not sure if this was implemented properly >.>

Reply:
Did you copy/paste the implementation for this from `data/scripts.js` and make the change on L296? If so this should be ok.